pyuvdata/uvcal/uvcal.py

```diff
@@ -1264,6 +1682,374 @@ def _check_freq_spacing(self, *, raise_errors=True):
+        cat_type : str
+            Type of phase center to be added. Must be one of:
+                "sidereal" (fixed RA/Dec),
+                "ephem" (RA/Dec that moves with time),
+                "driftscan" (fixed az/el position),
+                "unprojected" (no w-projection, equivalent to the old
+                `phase_type` == "drift").
+        cat_lon : float or ndarray
+            Value of the longitudinal coordinate (e.g., RA, Az, l) in radians of the
+            phase center. No default unless `cat_type="unprojected"`, in which case the
+            default is zero. Expected to be a float for sidereal and driftscan phase
+            centers, and an ndarray of floats of shape (Npts,) for ephem phase centers.
+        cat_lat : float or ndarray
+            Value of the latitudinal coordinate (e.g., Dec, El, b) in radians of the
+            phase center. No default unless `cat_type="unprojected"`, in which case the
+            default is pi/2. Expected to be a float for sidereal and driftscan phase
+            centers, and an ndarray of floats of shape (Npts,) for ephem phase centers.
+        cat_frame : str
+            Coordinate frame that cat_lon and cat_lat are given in. Only used
+            for sidereal and ephem targets. Can be any of the several supported frames
+            in astropy (a limited list: fk4, fk5, icrs, gcrs, cirs, galactic).
+        cat_epoch : str or float
+            Epoch of the coordinates, only used when cat_frame = fk4 or fk5. Given
+            in units of fractional years, either as a float or as a string with the
+            epoch abbreviation (e.g, Julian epoch 2000.0 would be J2000.0).
+        cat_times : ndarray of floats
+            Only used when `cat_type="ephem"`. Describes the time for which the values
+            of `cat_lon` and `cat_lat` are calculated, in units of JD. Shape is (Npts,).
+        cat_pm_ra : float
+            Proper motion in RA, in units of mas/year. Only used for sidereal phase
+            centers.
+        cat_pm_dec : float
+            Proper motion in Dec, in units of mas/year. Only used for sidereal phase
+            centers.
+        cat_dist : float or ndarray of float
+            Distance of the source, in units of pc. Only used for sidereal and ephem
+            phase centers. Expected to be a float for sidereal and driftscan phase
+            centers, and an ndarray of floats of shape (Npts,) for ephem phase centers.
+        cat_vrad : float or ndarray of float
+            Radial velocity of the source, in units of km/s. Only used for sidereal and
+            ephem phase centers. Expected to be a float for sidereal and driftscan phase
+            centers, and an ndarray of floats of shape (Npts,) for ephem phase centers.
+        info_source : str
+            Optional string describing the source of the information provided. Used
+            primarily in UVData to denote when an ephemeris has been supplied by the
```

Comment:
UVData -> UVCal

Reply:
Fixed in 766e32646768181b70ddd8d2e576d90ca57c254c.